BankManagerUIApp/BankManagerUIApp/Controller/BankViewController.swift

```diff
@@ -0,0 +1,283 @@
+        let button = UIButton()
+        button.setTitle(Namespace.reset, for: .normal)
+        button.setTitleColor(.systemRed, for: .normal)
+        
+        return button
+    }()
+    
+    private let timerLabel: UILabel = {
+        let label = UILabel()
+        
+        label.textAlignment = .center
+        
+        return label
+    }()
+    
+    private let statusLabelStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.distribution = .fillEqually
+        
+        let waitingLabel = UILabel()
+        waitingLabel.text = Namespace.waiting
+        waitingLabel.backgroundColor = .systemGreen
+        waitingLabel.textColor = .white
+        waitingLabel.textAlignment = .center
+        waitingLabel.font = UIFont.preferredFont(forTextStyle: .title1)
+        
+        let processingLabel = UILabel()
+        processingLabel.text = Namespace.processing
+        processingLabel.backgroundColor = .systemBlue
+        processingLabel.textColor = .white
+        processingLabel.textAlignment = .center
+        processingLabel.font = UIFont.preferredFont(forTextStyle: .title1)
+        
+        stackView.addArrangedSubview(waitingLabel)
+        stackView.addArrangedSubview(processingLabel)
+        
+        return stackView
+    }()
+    
+    private let customerStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .top
+        
+        stackView.distribution = .fillEqually
+        
+        return stackView
+    }()
+    
+    private let waitingStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = SpacingSetting.customerStackView
+        
+        return stackView
+    }()
+    
+    private let processingStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = SpacingSetting.customerStackView
+        
+        return stackView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemBackground
+        bank.delegate = self
+        configureUI()
+        resetUI()
+    }
+    
+    @objc private func didTapAddCustomer() {
+        guard let isTimerOn = timer?.isValid else {
+            return
+        }
+
+        if !isTimerOn {
+            startWorkTime = Date() - timeInterval
+            timer = Timer.scheduledTimer(timeInterval: Configuration.timeIntervalRepeat,
+                                         target: self,
+                                         selector: #selector(updateTimerLabel),
+                                         userInfo: nil,
+                                         repeats: true)
+        }
+        
+        bank.work(totalCustomer: Configuration.addCustomerAmount)
+    }
+    
+    @objc private func didTapReset() {
```

Comment:
```suggestion
    @objc private func didTapResetButton() {
```
```suggestion
    @objc private func resetButtonTapped() {
```

어떤 UIComponent가 눌렸는지까지 명시해주는게 좋을 것 같아요

Reply:
명시해줬습니다ㅎㅎ